Weather/Weather/Utils/Constants.swift

```diff
@@ -24,4 +24,13 @@ struct Constants {
     static let invalidRequest = "INVALID_REQUEST"
     static let blank = ""
     static let warning = "Warning"
+    
+    struct Screen {
+        static let widthScreen = UIScreen.main.bounds.width
+        static let heightScreen = UIScreen.main.bounds.height
+    }
+    
+    struct Storyboard {
```

Comment:
Để ra một file riêng em nhé
```suggestion
    struct Storyboard {
           static let main = UIStoryboard(name: "Main", bundle: nil)
    }
```